src/Symfony/Component/Config/Resource/DirectoryResource.php

```diff
@@ -18,7 +18,7 @@
  *
  * @final
  */
-class DirectoryResource implements SelfCheckingResourceInterface
+class DirectoryResource implements \Stringable, SelfCheckingResourceInterface
```

Comment:
I updated the PR after running `php-cs-fixer fix --rules=stringable_for_to_string`
This looks like a false-positive, isn't it? `SelfCheckingResourceInterface` extends `ResourceInterface`, which already extends `Stringable`, so that this change isn't necessary.
Something to improve on php-cs-fixer, right?

Reply:
> I suppose the issue is that php-cs-fixer doesn't run any code / load any classes.
> Maybe this one is doable using only tokens?

Indeed - Fixer runs on single file contextlessly (no knowledge about other file of project or of vendor/, while processing given file).

Teaching it to get such context (and basically evaluate file in context of whole project + deps) is possible, but changing architecture of Fixer deeply. Not a current prio - yet we are open for RFC and proposal of implementation.